app/controllers/games_controller.rb

```diff
@@ -33,6 +33,13 @@ def update
     handle_update_errors
   end
 
+  def move
+    respond_to do |format|
+      format.json { redirect_to piece_path(params[:piece_id]) }
+      format.html { redirect_to game_path(current_game) }
```

Comment:
Kicks user back to game page if they try to type in the url manually in the browser